src/ao_event.lua

```diff
@@ -64,6 +64,9 @@ local function AOEvent(initialData)
 					.. type(value)
 					.. ". Supported types are string, number, boolean, or nil."
 			)
+			if type(value) == "table" then
+				print("Invalid field value: " .. require("json").encode(value))
```

Comment:
We shouldn't require json inline like this. I believe it has bad memory usage characteristics. Let's factor it out to a local in the file.